doc/user/bgp.rst

```diff
@@ -2236,8 +2236,8 @@ attribute.
 Displaying Routes by AS Path
 ----------------------------
 
-.. index:: show bgp ipv4|ipv6 regexp LINE
-.. clicmd:: show bgp ipv4|ipv6 regexp LINE
+.. index:: show bgp ipv4|ipv6 regexp (1234567890_^|[,{}() ]$*+.?-\\)
+.. clicmd:: show bgp ipv4|ipv6 regexp (1234567890_^|[,{}() ]$*+.?-\\)
```

Comment:
Please keep it as `LINE` but document the string in the description below. And no need to escape the \ here I think